app/controllers/landing_controller.rb

```diff
@@ -0,0 +1,4 @@
+class LandingController < ApplicationController
```

Comment:
Controller names should be set in plural

Reply:
We kinda have the landing page controller set to singular in our course documentation since it's not linked to a model

Reply:
Okay